test/test_meta.py

```diff
@@ -0,0 +1,7 @@
+"""Tests that ensure fixtures act properly."""
+
+
+def test_generated_workspace(handled_workspace, server):
```

Comment:
Just having an argument named `handled_workspace` is enough to trigger the fixture?

Reply:
Yep, that's the beauty of pytest.